components/suggestion/emoticon_provider.jsx

```diff
@@ -81,7 +82,7 @@ export default class EmoticonProvider {
                 if (emoji.aliases) {
                     // This is a system emoji so it may have multiple names
                     for (const alias of emoji.aliases) {
-                        if (alias.indexOf(partialName) !== -1) {
+                        if (!EMOJI_CATEGORY_SUGGESTION_BLACKLIST.includes(emoji.category) && alias.indexOf(partialName) !== -1) {
```

Comment:
I could use some opinions about whether there is a better place to be filtering-out these suggestions.